using-mobile-extensions/adobe-campaign-standard/README.md

```diff
@@ -54,6 +54,13 @@ The **Request Timeout** value must be a non-zero number.
 This Campaign Standard extension requires the [Mobile Core](https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/tree/22148dec688a3240fc58fe5c7b0d5475f5940126/using-mobile-extensions/adobe-campaign-standard/mobile-core/README.md) and [Profile](https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/tree/22148dec688a3240fc58fe5c7b0d5475f5940126/using-mobile-extensions/adobe-campaign-standard/profile/README.md) extensions.
 {% endhint %}
 
+{% hint style="warning" %}
```

Comment:
Let's move this to section: https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/blob/d9c7b0f397ba04d9013365b7f209ef30619ac330/using-mobile-extensions/adobe-campaign-standard/README.md#set-up-in-app-messaging